finalPrep/modZeroHero.js

```diff
@@ -0,0 +1,176 @@
+var specialAbility = "Vacuum Mouth"
+
+
+// Declare two variables - greeting AND catchphrase
+//   greeting should be assigned to a string that uses concatenation to include the heroName
+//   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
+var greeting = "Hi, I'm your friendly, neihborhood " + heroName
+
+var catchphrase = `Looks like a job for my ${specialAbility}!`
+
+console.log(greeting);
+console.log(catchphrase);
+
+
+// Declare two variables - power AND energy - set to integers
+var power = 5
+var energy = 10
+
+// Declare two variables - fullPower AND fullEnergy
+//   fullPower should multiply your current power by 500
+//   fullEnergy should add 150 to your current energy
+
+var fullPower = (power * 500)
+var fullEnergy = (energy + 150)
+console.log(fullPower);
+console.log(fullEnergy);
+
+// Declare two variables - isHuman and identityConcealed - assigned to booleans
+var isHuman = false
+var identityConcealed = true
+
+// Declare two variables - archEnemies AND sidekicks
+//   archEnemies should be an array of at least 3 different enemy strings
+//   sidekicks should be an array of at least 3 different sidekick strings
+var archEnemies = ["Vacuum Cleaner", "Roomba", "Broom"]
+var sidekicks = ["Zapatito", "Teddy", "Charlie"]
+
+console.log(archEnemies);
+console.log(sidekicks);
+
+// Print the first sidekick to your console
+console.log(sidekicks[0])
+
+// Print the last archEnemy to the console
+console.log(archEnemies[2])
+// Write some code to add a new archEnemy to the archEnemies array
+archEnemies.push("Fire Alarm");
+
+// Print the archEnemies array to console to ensure you added a new archEnemy
+console.log(archEnemies);
+
+// Remove the first sidekick from the sidekicks array
+archEnemies.shift();
+
+// Print the sidekicks array to console to ensure you added a new sidekick
+console.log(archEnemies);
+// Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
+//   - dangerLevel should be an integer
+//   - saveTheDay should be a string a hero would say once they save the day
+//   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
+
+function assessSituation(dangerLevel, saveTheDay, badExcuse) {
+  console.log(assessSituation)
+}
+  var dangerLevel = 20
+  var saveTheDay = "I did it again!"
+  var badExcuse = "This is not good.....I'm gonna go home."
+
+
+// Your function should include an if/else statement that meets the following criteria
+//   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the badExcuse to the console
+//   - Anything dangerLevel that is between 10 and 50 should result in printing the saveTheDay string to the console
+//   - If the dangerLevel is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the console.
+
+if (dangerLevel > 50 ) {
```

Comment:
I expected to see this if/else statement _written within the function_ you defined above (the `assessSituation` function).  Like so:
```
function assessSituation(dangerLevel, saveTheDay, badExcuse) {
  if (dangerLevel > 50) {
      console.log(badExcuse);
  } else if (dangerLevel >= 10 && dangerLevel < 50 ){
    console.log(saveTheDay);
  } else{
    console.log("Meh. Hard pass.")
  }
}
```